Media/MediaEvent/schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Media/MediaEvent/schema.json",
+  "title": "FIWARE - Media / Media Event",
+  "description": "Base for all events raised by elements in the Kurento media server",
```

Comment:
see general comment on referring to specific technologies

Reply:
Done!